docs/source/api/models/lithium_ion/ecm_split_ocv.rst

```diff
@@ -0,0 +1,7 @@
+Equivalent Circuit Model with Split OCV (ECMsplitOCV)
+=====================================================
+
+.. autoclass:: pybamm.lithium_ion.ECMsplitOCV
```

Comment:
I think `pybamm.lithium_ion.SplitOCVR` would be more in line with what people would call this model

Reply:
Updated.